playbooks/rds_backup_monitoring_setup.yml

```diff
@@ -0,0 +1,168 @@
+- name: Prepare controller variables
+  import_playbook: setup_scenarios_controller.yml
+
+- name: Prepare variables
+  hosts: localhost
+  environment:
+    OS_CLIENT_CONFIG_FILE: "{{ clouds_yaml_file }}"
+    TF_VAR_network_id: "{{ main_network_id }}"
+    TF_VAR_router_id: "{{ main_router_id }}"
+    TF_VAR_subnet_id: "{{ main_subnet_id }}"
+    TF_VAR_controller_ip: "{{ controller_public_ip }}"
+  vars:
+    scenario_name: rds_backup_monitoring
+    terraform_base_dir: terraform
+    requirements: requirements.txt
+    terraform_workspace: prod
+    key_name: key_csm_controller
+    infra_state: present
+  roles:
+    - build_infrastructure
+  tasks:
+    - name: Register build result
+      set_fact:
+        ecs_instance_ip: "{{ tf_output.outputs['ecs_ip'].value }}"
+        rds_instance_id: "{{ tf_output.outputs['rds_id'].value }}"
+        db_password: "{{ tf_output.outputs['db_password'].value }}"
+        db_username: "{{ tf_output.outputs['db_username'].value }}"
+        db_address: "{{ tf_output.outputs['db_address'].value }}"
+        db_host: "{{ tf_output.outputs['db_host'].value }}"
+        db_port: "{{ tf_output.outputs['db_port'].value }}"
+    - name: Register ecs
+      add_host:
+        name: "ecs"
+        groups: "gatewayed"
+        ansible_host: "{{ ecs_instance_ip }}"
+        ansible_ssh_user: "linux"
+        ansible_ssh_private_key_file: "{{ local_private_key }}"
+        ansible_ssh_common_args: >
+          -o ProxyCommand='ssh -W %h:%p -q linux@{{ controller_public_ip }}
+          -i {{ local_private_key }}' -o PasswordAuthentication=no
+
+- name: Wait for host
+  hosts: ecs
+  gather_facts: no
+  tasks:
+    - name: Wait for host to be up
+      wait_for_connection:
+        delay: 5
+        timeout: 120
+
+- name: Start Server
+  hosts: ecs
+  become: yes
+  vars:
+    server_port: 8081
+    run_option: "sqla"
+    drivername: "postgresql+psycopg2"
+  roles:
+    - role: too_simple_server
+      vars:
+        pg_db_url: "{{ hostvars.localhost.db_address }}"
+        pg_database: "entities"
+        pg_username: "{{ hostvars.localhost.db_username }}"
+        pg_password: "{{ hostvars.localhost.db_password }}"
+        pg_host: "{{ hostvars.localhost.db_host }}"
+        pg_port: "{{ hostvars.localhost.db_port }}"
+        debug: True
+  tasks:
+    - name: Register facts
+      set_fact:
+        server_local_port: "{{ server_port }}"
+        run_option: "{{ run_option }}"
+        drivername: "{{ drivername }}"
+        source: "/tmp/rds/{{ run_option }}/data.yaml"
+
+    - name: Update and upgrade apt packages
+      apt:
+        upgrade: yes
+        update_cache: yes
+
+    - name: Install PIP, rsync libpq-dev and python-dev
+      apt:
+        name:
+          - python3-setuptools
+          - python3-pip
+          - libpq-dev
+          - python-dev
+          - cron
+          - rsync
+
+    - name: Install pg2 and SQLAlchemy for Postgresql
+      pip:
+        executable: pip3
+        name:
```

Comment:
It's better to move python requirements to `requirements.txt`, e.g. `rds-requirements.txt`